specification/security/resource-manager/Microsoft.Security/preview/2023-09-01-preview/securityConnectorsDevOps.json

```diff
@@ -0,0 +1,3785 @@
+          "type": "string",
+          "readOnly": true
+        },
+        "fullyQualifiedFriendlyName": {
+          "description": "Gets or sets the human readable fully-qualified name of the Project object.\r\n\r\nThis contains the entire namespace hierarchy as seen on GitLab UI where entities are separated by the '/' character.",
+          "type": "string",
+          "readOnly": true
+        },
+        "fullyQualifiedParentGroupName": {
+          "description": "Gets or sets the fully-qualified name of the project's parent group object.\r\n\r\nThis contains the entire hierarchy where namespaces are separated by the '$' character.",
+          "type": "string",
+          "readOnly": true
+        },
+        "url": {
+          "description": "Gets or sets the Url of the GitLab Project.",
+          "type": "string",
+          "readOnly": true
+        },
+        "onboardingState": {
+          "$ref": "#/definitions/OnboardingState"
+        }
+      }
+    },
+    "InheritFromParentState": {
+      "description": "Update Settings.\r\n\r\nEnabled - Resource should inherit configurations from parent.\r\nDisabled - Resource should not inherit configurations from parent.",
+      "enum": [
+        "Disabled",
+        "Enabled"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "InheritFromParentState",
+        "modelAsString": true,
+        "modelAsExtensible": true
+      }
+    },
+    "OnboardingState": {
+      "description": "Details about resource onboarding status across all connectors.\r\n\r\nOnboardedByOtherConnector - this resource has already been onboarded to another connector. This is only applicable to top-level resources.\r\nOnboarded - this resource has already been onboarded by the specified connector.\r\nNotOnboarded - this resource has not been onboarded to any connector.\r\nNotApplicable - the onboarding state is not applicable to the current endpoint.",
+      "enum": [
+        "NotApplicable",
+        "OnboardedByOtherConnector",
+        "Onboarded",
+        "NotOnboarded"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "OnboardingState",
+        "modelAsString": true,
+        "modelAsExtensible": true
+      }
+    },
+    "ProvisioningState": {
+      "description": "ProvisioningState of resource.\r\n\r\nPending - Provisioning pending.\r\nFailed - Provisioning failed.\r\nSucceeded - Successful provisioning.\r\nCanceled - Provisioning canceled.\r\nPendingDeletion - Deletion pending.\r\nDeletionSuccess - Deletion successful.\r\nDeletionFailure - Deletion failure.",
+      "enum": [
+        "Succeeded",
+        "Failed",
+        "Canceled",
+        "Pending",
+        "PendingDeletion",
+        "DeletionSuccess",
+        "DeletionFailure"
+      ],
+      "type": "string",
+      "x-ms-enum": {
```

Comment:
This will conflict with autorest generation with another ProvisioningState already defined:

FATAL: System.InvalidOperationException: Swagger document contains two or more x-ms-enum extensions with the same name 'ProvisioningState' and different values: Succeeded,Failed,Canceled,Pending,PendingDeletion,DeletionSuccess,DeletionFailure vs. Succeeded,Failed,Updating

renaming the definition to ResourceProvisioningState did help to resolve this conflict

Reply:
Might need input from @ivadim on this.